evm/bridge-token-factory/contracts/BridgeTokenFactory.sol

```diff
@@ -55,6 +55,7 @@ contract BridgeTokenFactory is
 
     function deployToken(bytes calldata signatureData, BridgeTypes.MetadataPayload calldata metadata) payable external returns (address) {
         bytes memory borshEncoded = bytes.concat(
+            bytes("\x01"),
```

Comment:
Should we have an enum instead?

Reply:
6d94fd3f342218bab869f1623a8edc5ba03506dc